Yoast/wordpress-seo
# FRO-95 Add namespace to CSS variables

## Context


* Add namespacing to the CSS variables to resolve conflicts with themes

## Summary


This PR can be summarized in the following changelog entry:

* Fixes a bug where our global CSS variables could conflict with global CSS variables of themes

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Pull this branch
* Pull the release/14.3 branch of the javascript repo
* Link the monorepo and build the plugin
* Check if the styling still is correct of the plugin

## UI changes

## Documentation

## Quality assurance


Fixes https://yoast.atlassian.net/browse/FRO-95
